packages/trader/src/Modules/Trading/Helpers/video-config.ts

```diff
@@ -14,22 +14,28 @@ export const DESCRIPTION_VIDEO_ID: TDtraderVideoUrl = {
         light: 'c1d64d5ffaf449e3326d387a70621a4c',
         dark: '9657e4adcfd9274d41a042b7b6c42d60',
     },
-
     high_low: {
         light: 'f528c662c377601908a741ae2aedaec6',
         dark: '4c05445e1bd28f3f9f95b0ab566c4f9b',
     },
+    match_diff: {
+        light: '8693a02397d284ffcf0608e4fda702d9',
+
+        dark: '2e56ad7ea67bf3f9a869de6336df1796',
+    },
+    over_under: {
+        light: '9e2be907cba9b38352890e52cfd8cbaf',
 
```

Comment:
```suggestion
```

Reply:
yes, i removed the unnecessary line breaks)